parsers/MN.py

```diff
@@ -107,13 +108,16 @@ def fetch_production(
     prod_mix.add_value("coal", data["coal"])
     prod_mix.add_value("solar", data["solar"])
     prod_mix.add_value("wind", data["wind"])
+    storage_mix = StorageMix()
+    storage_mix.add_value("battery", data["battery_storage"])
```

Comment:
I think we need to invert the value here as we treat negative as discharge and positive as charging.